netci.groovy

```diff
@@ -2515,6 +2575,46 @@ combinedScenarios.each { scenario ->
                                         }
                                     }
                                 }
+                                else if ( architecture == 'arm' && ( os == 'Ubuntu' || os == 'Ubuntu16.04' || os == 'Tizen')) {
+                                    // Cross build for ubuntu-arm, ubuntu16.04-arm and tizen-armel
+                                    // Define the Windows Tests and Corefx build job names
+                                    def WindowTestsName = projectFolder + '/' +
+                                                          Utilities.getFullJobName(project,
+                                                                                   getJobName(lowerConfiguration,
+                                                                                              'x64' ,
+                                                                                              'windows_nt',
+                                                                                              'default',
+                                                                                              true),
+                                                                                   false)
+                                    def corefxFolder = Utilities.getFolderName('dotnet/corefx') + '/' +
+                                                       Utilities.getFolderName(branch)
+
+                                    // Copy the Windows test binaries and the Corefx build binaries
+                                    copyArtifacts(WindowTestsName) {
```

Comment:
Yes,  we copy (1) test binary and (2) corefx binary from the latest to prepare coreclr test.

Because I don't know well about relation between coreclr CI and corefx CI, I just  followed a way of current CI for arm and other architectures.
https://github.com/dotnet/coreclr/blob/master/netci.groovy#L2505-L2516 (for arm)
https://github.com/dotnet/coreclr/blob/master/netci.groovy#L2895-L2900 (for other)

If this is not a right way, please let me know how to fetch correct corefx and test for coreclr testing.